KmsClient.js

```diff
@@ -440,26 +498,34 @@ export class KmsClient {
       capability = _getRootZcapId({keyId});
     }
 
+    let headers;
     try {
       // sign HTTP header
-      const headers = await signCapabilityInvocation({
+      headers = await signCapabilityInvocation({
         url, method: 'post', headers: this.defaultHeaders,
         json: operation, capability, invocationSigner,
         capabilityAction: 'verify'
       });
+    } catch(e) {
+      _handleClientError({
+        message:
+          'Error signing zCap invocation during "verify" operation.',
+        cause: e
+      });
```

Comment:
This approach seems a bit odd to me. Could we instead wrap the whole function in a try/catch -- and always catch the error, generate a new one that says `Error during "verify" operation.` and then just set the cause of that error to be whatever was thrown? Then there's no need to call `_handleClientError` in multiple places -- and maybe we don't even need any special logic, etc.?

Reply:
@dlongley - that was my initial approach (wrapping whole function in a try/catch). But I also want to differentiate when the error is caused during zcap signing, or during the WebKMS operation itself. Hence the two separate usages.

Reply:
Wouldn't the error raised via `signCapabilityInvocation` indicate that the problem was a zcap signing one? If not, could we use a helper that wraps `signCapabilityInvocation` and throws a more specific error -- and then just call that helper function in place of `signCapabilityInvocation` everywhere, wrap everything in try/catch, and save the repetition that way?

Reply:
I took the approach you suggested (wrapped the sign call into a helper function, with a more specific error message).